doc/developer_guide/developers.rst

```diff
@@ -363,12 +363,21 @@ However, the necessary nodes in the new tree structure will be annotated
 with information to enable the original language-specific syntax to be
 recreated if required.
 
+
+Branching construct
+-------------------
+
 .. autoclass:: psyclone.psyGen.IfBlock
     :members:
 
+
+Iteration construct
+-------------------
+
 .. autoclass:: psyclone.psyGen.Loop
```

Comment:
This is nice but does reveal some errors in our Sphinx docstrings. In particular, the description of `unique_modified_args` is not formatted correctly and the entry for `reference_accesses` says that the loop variable is marked as READWRITE when I think (although I might be wrong) that it is marked as READ+WRITE. @hiker can probably advise here.

Reply:
fixed these 2 issues with the docstrings

Reply:
Yes, READ+WRITE is now correct, I didn't fix up the documentation, so thanks. Though READ+WRITE is only used in str() to indicate an read and write access at the same location. So maybe this should say that the loop variable is marked with a READ and a WRITE access at the same location?

Reply:
Thanks @hiker and @sergisiso.